pkg/autoscaler/autoscaler.go

```diff
@@ -195,14 +206,15 @@ func (a *Autoscaler) Scale(ctx context.Context, now time.Time) (desiredPodCount
 	case a.deciderSpec.TargetBurstCapacity >= 0:
 		excessBC = int32(math.Floor(float64(originalReadyPodsCount)*a.deciderSpec.TotalValue - observedStableValue -
 			a.deciderSpec.TargetBurstCapacity))
-		logger.Debugf("PodCount=%v TotalConc=%v ObservedStableConc=%v TargetBC=%v ExcessBC=%v",
+		logger.Debugf("PodCount=%v TotalValue=%v ObservedStableValue=%v TargetBC=%v ExcessBC=%v",
```

Comment:
Add here _which_ metric we're reporting.
That would be the benefit of putting it as the `logger.With()` since it will automatically annotate the _kind_ of metric we're using in all statements.

Reply:
Now I think adding metric to the logger as annotation makes more sense if more than one place needs the metric information. Added above.